docs/whats-new.asciidoc

```diff
@@ -74,5 +74,16 @@ image::whats-new/images/8.1/cases-kpis.png[]
 
 In addition to improvements to indicator match rule performance, the default indicator index query for custom and prebuilt {security-guide}/rules-ui-create.html#create-indicator-rule[indicator match rules] has been updated to `@timestamp > "now-30d/d"` to fine-tune the query range and performance.
 
+*Deploy DGA and Living-off-the-land supervised models in Fleet*
+ 
+Incorporating supervised models into integration packages provides you a more
+seamless experience to install package artifacts inside {kib} with a single
+click. Now you can deploy
+https://www.elastic.co/blog/supervised-and-unsupervised-machine-learning-for-dga-detection[DGA]
```

Comment:
Since that's the old 2020 blog, I think it's better to link to the new integration docs:
```suggestion
https://docs.elastic.co/en/integrations/dga[DGA]
```